packages/expo/src/KeepAwake/index.js

```diff
@@ -1,7 +1,5 @@
 import React from 'react';
-import { NativeModules } from 'react-native';
-
-const { ExponentKeepAwake } = NativeModules;
+import ExponentKeepAwake from './ExponentKeepAwake';
```

Comment:
```suggestion

import ExponentKeepAwake from './ExponentKeepAwake';
```

Separate node_modules imports from local imports: https://github.com/expo/expo/blob/master/guides/Expo%20JavaScript%20Style%20Guide.md#imports